pkg/cmd/datastore/datastore.go

```diff
@@ -53,6 +54,7 @@ func DefaultReadConnPool() *ConnPoolConfig {
 		MaxOpenConns:        20,
 		MinOpenConns:        20,
 		HealthCheckInterval: 30 * time.Second,
+		MaxLifetimeJitter:   10 * time.Minute,
```

Comment:
Do we want to explicitly not set this here and let it use the 20% default when unset?